app/templates/components/upcoming-project-card.hbs

```diff
@@ -98,10 +98,8 @@
                     @date={{milestone.dcpActualenddate}}
                   />
                 </small>
-              {{#if (or (not milestone.dcpActualenddate) (is-before milestone.dcpActualenddate))}}
-                  {{hearings-list-for-milestones-list milestone=milestone}}
-              {{/if}}
               {{/if}}
+              {{hearings-list-for-milestones-list milestone=milestone}}
```

Comment:
change request: We should use Glimmer component (bracket) syntax, like in reviewed-project-card